platforms/common/Serial.cpp

```diff
@@ -130,6 +135,24 @@ bool Serial::setFlowcontrol(FlowControl flowcontrol)
 	return _impl.setFlowcontrol(flowcontrol);
 }
 
+bool Serial::getSingleWireMode() const
+{
+	return _impl.getSingleWireMode();
+}
+bool Serial::setSingleWireMode()
+{
+	return _impl.setSingleWireMode();
+}
+
+bool Serial::getSwapRxTxMode() const
+{
+	return _impl.getSwapRxTxMode();
+}
+bool Serial::setSwapRxTxMode()
```

Comment:
Can we also include the prototypes for inverted mode?
The idea is when introducing a new interface you try to accommodate for all uses and write these prototypes out.

Reply:
Sure, I just added them.